src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Square implements Figure {
+    private int side;
+    private String color;
+    public Square() {
+        this.side = side;
+        this.color = color;
```

Comment:
The constructor of `Square` is not correctly implemented. It should take parameters for `side` and `color`, but currently, it's using uninitialized fields which leads to assigning default values (0 and null). You need to add parameters to the constructor and use them to initialize the fields.